MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -180,13 +180,19 @@ public bool NeedsRebuild(PipelineBuildEvent cachedEvent)
                 cachedEvent.DestFile != DestFile)
                 return true;
 
+            // Did the importer assembly change?
+            if (manager.GetImporterAssemblyTimestamp(cachedEvent.Importer) > cachedEvent.DestTime)
```

Comment:
You know...  I think we might need to keep track of the assembly timestamps explicitly here instead of using the `DestTime`.  You could have a case where the assembly is rolled back to an older version with an older timestamp.